website/src/content/development/notifications.mdx

```diff
@@ -0,0 +1,343 @@
+---
+title: Notifications
+---
+
+In the previous sections, we learned how to develop forms. In this section we focus on providing users with relevant information in response to a successful or failed form submission.
+
+To facilitate the use of notifications, Custom Applications come with a built-in [notification system](https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-actions-global#notification-system).
+
+# Notification system
+
+Notifications are an important way of providing feedback in response to user actions.
+Submission of a form might end with a success, but for various reasons it might also fail.
+Either way, a user should be informed what was the end result of the action.
+
+The Custom Application built-in notification system provides a way of showing notifications by dispatching Redux actions throught bound action creators or React hooks exposed from the [`@commercetools-frontend/actions-global`](/api-reference/commercetools-frontend-actions-global) package.
+Notifications can be dispatched in one of the predefined placement options (referred to as **domains**, for instance `page` or `side`) and in one of various kinds depending on the domain (for instance `error`, `warning` or `success`).
```

Comment:
I wouldn't go much into the technical details but give a general overview of how it works.

```suggestion
Notifications are grouped by what we call "domains". A domain indicates where the application should be displayed on the page.

* `page`: Page notifications are displayed on top of the content page.
* `side`: Side notifications are smaller and they slide in from the right side of the page.

Notifications are also of different types: `success`, `error`, `warning`, `info`. Depending on the specific use case you would choose the appropriate type. For example, if a form has been successfully submitted, you would show a "side" notification of type `success`.

Triggering notifications is done by dispatching global actions. Please refer to the package [`@commercetools-frontend/actions-global`](/api-reference/commercetools-frontend-actions-global) for implementation details.
```